### PR TITLE
issues/634: `orientdb.profile=orientdb_MEMORY` for `jetty-block` profile

### PR DESCRIPTION
**Pull request by sbespalov**

### CODE REVIEW

**Comment by sbespalov** [comment]
@carlspring it's ready to be merged

File: strongbox-web-core/pom.xml [hunk 1]
```diff
@@ -906,7 +906,18 @@
                                     <name>strongbox.truststore.jks</name>
                                     <value>${dir.strongbox.home}/etc/ssl/truststore.jks</value>
                                 </systemProperty>
-
+                                <systemProperty>
+                                    <name>orientdb.profile</name>
```
Comment (carlspring) [inline strongbox-web-core/pom.xml]: Could you please prefix this with `strongbox.*`?

Comment (sbespalov) [reply strongbox-web-core/pom.xml]: ok

Comment (sbespalov) [reply strongbox-web-core/pom.xml]: done

File: strongbox-data-service/src/main/java/org/carlspring/strongbox/config/ConnectionConfigOrientDB.java [hunk 2]
```diff
@@ -116,26 +115,18 @@ public void setPassword(String password)
...
     {
-        if (System.getProperty(PROPERTY_PROFILE) != null)
-        {
-            LOGGER.info(String.format("Can't override already provided OrientDB connection profile, skip profile [%s] bootstrap.",
-                                      profile));
-            return;
-        }
-
+        String profile = System.getProperty(PROPERTY_PROFILE, PROFILE_MEMORY);
         LOGGER.info(String.format("Bootstrap OrientDB connection properties with profile [%s].",
```
Comment (carlspring) [inline strongbox-data-service/src/main/java/org/carlspring/strongbox/config/ConnectionConfigOrientDB.java]: Also, the names of loggers are all lowercase in our project (`logger`) everywhere across the codebase.

Comment (sbespalov) [reply strongbox-data-service/src/main/java/org/carlspring/strongbox/config/ConnectionConfigOrientDB.java]: done

**Comment by carlspring** [comment]
Thanks!

**Pull request merged**